foxtrot-server/src/main/java/com/flipkart/foxtrot/server/resources/ClusterHealthResource.java

```diff
@@ -64,7 +75,17 @@ public NodesStatsResponse getNodeStat() throws ExecutionException, InterruptedEx
     @Timed
     @Path("indicesstats")
     @ApiOperation("getIndicesStat")
-    public IndicesStatsResponse getIndicesStat() throws ExecutionException, InterruptedException {
-        return queryStore.getIndicesStats();
```

Comment:
This is changed to send additional information like  avg document size, number of columns on the cluster info page